test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,37 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace {
+const std::shared_ptr<Envoy::Http::RequestHeaderMap>& getDefaultHeaderMap() {
+  static auto* header =
+      new std::shared_ptr<Envoy::Http::RequestHeaderMap>{new Envoy::Http::TestRequestHeaderMapImpl{
+          {":scheme", "http"}, {":method", "GET"}, {":path", "/"}, {":host", "localhost"}}};
+  return *header;
+}
+
+// Default function for request generator when the content doesn't matter.
+RequestGenerator getDefaultRequestGenerator() {
+  RequestGenerator request_generator = []() {
+    return std::make_unique<RequestImpl>(getDefaultHeaderMap());
+  };
+  return request_generator;
+}
+// Helper function to get headers in a set that should be verified during the test.
+std::string getPathFromRequest(const Envoy::Http::RequestHeaderMap& header) {
+  return std::string(header.getPathValue());
+}
+struct ClientSetupParameters {
```

Comment:
Documenting this struct, its constructor and members will help future readers / editors of the test.

Reply:
Done.